src/runtime_src/core/pcie/linux/scan.h

```diff
@@ -61,20 +61,22 @@ class pci_device {
         std::string& err_msg, std::vector<uint64_t>& iv);
     void sysfs_get(const std::string& subdev, const std::string& entry,
         std::string& err_msg, std::string& s);
-    void sysfs_get(const std::string& subdev, const std::string& entry,
-        std::string& err_msg, bool& b);
     void sysfs_get(const std::string& subdev, const std::string& entry,
         std::string& err_msg, std::vector<char>& buf);
     template <typename T>
     void sysfs_get(const std::string& subdev, const std::string& entry,
-        std::string& err_msg, T& i) {
+        std::string& err_msg, T& i, const int& default_val) {
```

Comment:
should we do const T& default_val?

Reply:
It is better to leave it as int. "T& i" and "T& default_val" don't use the same T, eg, first one is uint_64 and second one is simple int so it gives an error.